### DISCUSSION TITLE
下载editor导出的项目“旋转的小鸭子”后，在本地无法运行

### DISCUSSION THREAD

**Discussion by cnXinyaZ** [galacean/discussion#132] [Editor] [answered]
运行npm i和npm run dev后，打开网页显示 Failed to start project TypeError: Failed to construct 'URL': Invalid URL
![image](https://github.com/user-attachments/assets/8a73a98f-0d26-4bc2-ae39-7b6c57073cbb)
是需要改哪些配置吗？
在线project地址是https://galacean.antgroup.com/editor/project/63400012

**Comment by GuoLei1990** [comment, marked as answer]
BUG 已定位，感谢反馈！我们尽快热更修复

**Comment by Sway007** [comment]
是我们之前一个发布的小版本引擎引入的bug导致的，现在已帮您升级项目至当前最新版本1.3.24，重新下载项目安装运行即可。再次感谢您的反馈

**Comment by GuoLei1990** [comment]
具体升级步骤：

1. 
![image](https://github.com/user-attachments/assets/f8fd40f6-0eb8-4023-b779-671de7a08396)
2. 
![image](https://github.com/user-attachments/assets/22db8455-6c54-4c80-b94b-dbe48357a633)

Issue 暂时关闭，如仍有问题可重新打开！
